### PR TITLE
Complete missing step for working-with-anp

### PR DESCRIPTION
**Pull request by XiShanYongYe-Chang**
Signed-off-by: changzhen <changzhen5@huawei.com>

**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

Complete missing step for `working-with-anp.md`. After deploying ANP, we need to add command flags `cluster-api-endpoint` and `proxy-server-address` for karmada-agent deployment.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by XiShanYongYe-Chang** [comment]
/cc @Tingtal

Hi, can you help review it?

File: docs/working-with-anp.md [hunk 1]
```diff
@@ -292,4 +292,16 @@ Deploying the proxy-agent in the pull mode member cluster (in this example, clus
 kind load docker-image swr.ap-southeast-1.myhuaweicloud.com/karmada/proxy-agent:0.0.24 --name member3
 kubectl --kubeconfig=/root/.kube/members.config --context=member3 apply -f proxy-agent.yaml
 ```
-**The ANP deployment is complete.**
\ No newline at end of file
+**The ANP deployment is complete.**
+
+### Step 6: Add command flags for karmada-agent deployment
+
+After deploying ANP deployment, we need to add command flags `cluster-api-endpoint` and `proxy-server-address` for karmada-agent deployment on `member3` cluster.
```
Comment (RainbowMango) [inline docs/working-with-anp.md:299]:
```suggestion
After deployed ANP deployment, we need to add extra command flags `--cluster-api-endpoint` and `--proxy-server-address` for `karmada-agent` deployment on `member3` cluster.
```

Comment (XiShanYongYe-Chang) [reply docs/working-with-anp.md]: Thanks, I have updated it.

File: docs/working-with-anp.md [hunk 2]
```diff
@@ -292,4 +292,16 @@ Deploying the proxy-agent in the pull mode member cluster (in this example, clus
 kind load docker-image swr.ap-southeast-1.myhuaweicloud.com/karmada/proxy-agent:0.0.24 --name member3
 kubectl --kubeconfig=/root/.kube/members.config --context=member3 apply -f proxy-agent.yaml
 ```
-**The ANP deployment is complete.**
\ No newline at end of file
+**The ANP deployment is complete.**
+
+### Step 6: Add command flags for karmada-agent deployment
+
+After deployed ANP deployment, we need to add extra command flags `--cluster-api-endpoint` and `--proxy-server-address` for `karmada-agent` deployment on `member3` cluster.
```
Comment (Tingtal) [inline docs/working-with-anp.md:299]:
```suggestion
After deploying the ANP deployment, we need to add extra command flags `--cluster-api-endpoint` and `--proxy-server-address` for `karmada-agent` deployment in `member3` cluster.
```

File: docs/working-with-anp.md [hunk 3]
```diff
@@ -292,4 +292,16 @@ Deploying the proxy-agent in the pull mode member cluster (in this example, clus
...
+
+Where `cluster-api-endpoint` is the APIEndpoint of the cluster. You can obtain it from the KubeConfig file of the `member3` cluster. 
+
+Where `proxy-server-address` is the address of the proxy server that is used to proxy the cluster. In current case, we can set `proxy-server-address` to `http://<karmada_controlplan_addr>:8088`. Get `karmada_controlplan_addr` value through the following command:
+```shell
+docker inspect --format='{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' karmada-host-control-plane
+```
+Port `8088` is set by our code modification in ANP: https://github.com/mrlihanbo/apiserver-network-proxy/blob/v0.0.24/dev/cmd/server/app/server.go#L267, you can also modify it to a different value.
```
Comment (Tingtal) [inline docs/working-with-anp.md:307]:
```suggestion
Port `8088` is set by our code modification in ANP: https://github.com/mrlihanbo/apiserver-network-proxy/blob/v0.0.24/dev/cmd/server/app/server.go#L267. You can also modify it to a different value.
```

**Review by Tingtal — CHANGES_REQUESTED** [review]

**Comment by XiShanYongYe-Chang** [comment]
@Tingtal thanks for you review :)

File: docs/working-with-anp.md [hunk 4]
```diff
@@ -292,4 +292,16 @@ Deploying the proxy-agent in the pull mode member cluster (in this example, clus
...
 ```
-**The ANP deployment is complete.**
\ No newline at end of file
+**The ANP deployment is complete.**
+
+### Step 6: Add command flags for karmada-agent deployment
+
+After deploying the ANP deployment, we need to add extra command flags `--cluster-api-endpoint` and `--proxy-server-address` for `karmada-agent` deployment in `member3` cluster.
+
+Where `cluster-api-endpoint` is the APIEndpoint of the cluster. You can obtain it from the KubeConfig file of the `member3` cluster.
```
Comment (RainbowMango) [inline docs/working-with-anp.md:301]:
```suggestion
Where `--cluster-api-endpoint` is the APIEndpoint of the cluster. You can obtain it from the KubeConfig file of the `member3` cluster. 
```
Add `--` to all flags.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**